exampleCourse/questions/element/stringInput/server.py

```diff
@@ -35,3 +35,4 @@ def generate(data):
     data["correct_answers"]["ans6"] = e
     data["correct_answers"]["ans7"] = f
     data["correct_answers"]["ans8"] = '"Ee"'
+    data["correct_answers"]["ans9"] = "Hello\r\nWorld!"
```

Comment:
Actually, remember that `print` also adds a line break, so this should be:
```suggestion
    data["correct_answers"]["ans9"] = "Hello\r\nWorld!\r\n"
```

That said, it might be good to encourage somewhere (docs? example?) to consider also setting `ignore-leading-trailing` and `ignore-spaces` when using `multiline`.

Reply:
I'll do this in the docs, since I think we want the example question to show off the multiline behavior.

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/9438/commits/0eef7c6d48b3dff2c7d06f952f4ea4da6dfc59a8#diff-b75a5321e7d3f379d69d65e5a5e0b4deebb9fb34092e525ec7254ab78587fee0